app/presenters/check_answers/base.rb

```diff
@@ -0,0 +1,25 @@
+module CheckAnswers
+  class Base
+    attr_accessor :group, :section
+
+    def translate_table_key(table, key, **opt)
+      I18n.t("steps.check_answers.show.sections.#{table}.#{key}.key", **opt)
+    end
+
+    def title(**opt)
+      I18n.t("steps.check_answers.groups.#{group}.#{section}.title", **opt)
```

Comment:
same here
```
I18n.t("steps.check_answers.groups.#{group}.#{section}", **opt)
```

Reply:
Going to keep these the same in case there's other information that is going to be added at this level such as subtitle/hint text